### ISSUE TITLE
Task collection from whl file: Show in interface that this is possible

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-web#349]
I was planning to write an issue to allow this functionality. Turns out, it already works, it just wouldn't be obvious that one could try that from the interface.

----

Using the Fractal task package template (https://github.com/fractal-analytics-platform/fractal-tasks-template), we can now have users create their own Fractal task package. Expecting all those users to deploy those packages to PyPi would be a bit much though. Therefore, it would be very useful to expose the ability to collect tasks based on a whl file in Fractal web.

I currently do this using the CLI fractal-client:
```
fractal task collect /absolute/path/to/gliberal-scMultipleX/dist/scmultiplex-0.4.dev137+g4b46ca5-py3-none-any.whl --package-extras fractal-tasks
```

I tested doing this in Fractal web by using the existing "Collect tasks from a package" option. It basically works, surprisingly!

Basically, I enter the full path to the whl file under package and then click collect.

<img width="1391" alt="Screenshot 2023-11-29 at 14 35 05" src="https://github.com/fractal-analytics-platform/fractal-web/assets/18033446/477ab8c3-e5e2-4d6e-9128-9c9e5b4f023f">

2 limiting factors:
1. It's not clear to me yet what happens to the "Package Version" field in this context? e.g. if I'd set a version different from the package I'm using. To be verified.
2. The "Package" field can be used for that is not obvious (I didn't expect it to work from the web interface, but given the underlying fractal API, I can see why it does work)

**Comment by tcompa** [comment]
This method has been used for testing since the very beginning of fractal-web, and I thought it was known that both package name (for packages on PyPI) or on-disk package path are accepted. This was also meant to support non-public packages being developed internally (as I think some of the fractal-tasks-core-dependent ones were, in the past).

Anyway, let's make it more obvious. Would it be OK to have one of the typical "info" popover button next to "package"? The info message could be something like "The name of a package published on PyPI, or the full path to a wheel file".

---

The version setting is ignored, when using a local wheel file:
![image](https://github.com/fractal-analytics-platform/fractal-web/assets/3862206/ef9056ff-f5dd-4673-b229-990bf8f7a207)

This should probably appear as an API error, however: if the user provides a wheel file, then they cannot also ask for a version.

**Comment by jluethi** [comment]
> Would it be OK to have one of the typical "info" popover button next to "package"? The info message could be something like "The name of a package published on PyPI, or the full path to a wheel file"

Sounds good. And yes, we can go with that message :)

**Comment by zonia3000** [comment]
Closing, done in #364
